docs/commands/deprovision.md

```diff
@@ -16,6 +16,7 @@ Deletes a service instance.
 |--------|-----------|
 | -h, --help  Help for deprovision command.| No |
 | -f, --force Force delete - without confirmation | No |
+| --purge Delete this resource and all its related data | No |
```

Comment:
how is that different than deleting the resource without `--purge`?

Reply:
Changed to --force-delete with description by Daniel Korac